src/components/Book.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import './Book.css';
+import { removeBook } from '../redux/Books/books';
+
+const Book = ({ book }) => {
+  const dispatch = useDispatch();
+  const { id } = book;
+  const percentage = 66;
+
+  return (
+
+    <div className="bookContents">
+      <div className="content1">
+        <div className="content1FirstChild">
+          <p className="genre">{book.category}</p>
+          <h2 className="title">{book.title}</h2>
+          <p className="author">{book.author}</p>
+        </div>
+        <div className="content1SecondChild">
+          <button type="button" className="commentBtn">Comment</button>
+          <button type="button" className="removeBtn" onClick={() => dispatch(removeBook(id))}>Remove</button>
+          <button type="button" className="editBtn">Edit</button>
+        </div>
+      </div>
+      <div className="content2">
+        <div className="circular-progressbar">
+          <CircularProgressbar
+            className="progresBar"
+            value={percentage}
+          />
+        </div>
+        <div className="progressDetails">
+          <span className="percentage">
+            {`${percentage}%`}
+          </span>
+          <h2 className="completed">Completed</h2>
+        </div>
+
+      </div>
+      <div className="line" />
+
+      <div className="content3">
+        <span className="currentChapter">CURRENT CHAPTER</span>
+        <span className="chapter">Chapter 17</span>
+        <button type="button" className="updateBtn">UPDATE PROGRESS</button>
+      </div>
+    </div>
```

Comment:
- [ ] The book card has more height than the one in the Design please try to decrease the height and increase the width  of the card in order to match the design 
- Yours 
![image](https://user-images.githubusercontent.com/61790587/178990197-b46d4f4b-2ddd-4e15-992e-f4c1cd0cc5d8.png)
- Design 
![image](https://user-images.githubusercontent.com/61790587/178990314-7d25d7ce-6512-42eb-a6b1-4ea80d8f381c.png)